fellowship.js

```diff
@@ -19,30 +19,63 @@ var fellowship = {
   makeMiddleEarth: function(lands) {
     console.log(lands)
     // create a section tag with an id of middle-earth
+    var earth = document.createElement("section");
+    earth.id="middle-earth";
     // add each land as an article tag
+    for (var i=0; i<lands.length; i++) {
+      var landarticle= document.createElement("article");
     // inside each article tag include an h1 with the name of the land
+    var h1tag=document.createElement("h1");
```

Comment:
make sure to keep your indentation consistent, it's hard to read when the indentation jumps around!